### PR TITLE
ExomeDepth singularity support

### PR DESCRIPTION
**Pull request by rernst**
- Use [rocker tidyverse](https://hub.docker.com/layers/rocker/tidyverse/3.5.1/images/sha256-916d4e919fbac9ee1f06db7622b4731268dff0cb17998b3db6f629a3e58f73a7?context=explore) docker container to run R.
- Replace `module load` with `singularity exec` from system commands.
- Use genome reference from settings and only add filename to vcf file.
- Remove hard-coded reference path from `template.vcf`.

### CODE REVIEW

**Comment by rernst** [comment]
@melferink We need to reinstall all R packages and set `r_library_path` in `settings.py`

File: ExomeDepth/README.md [hunk 1]
```diff
@@ -1,14 +1,32 @@
...
-``` bash
-virtualenv -p python3 venv
-. venv/bin/activate
+## Installation 
+To run the ExomeDepth wrapper scripts we need to create a virtual python environment and folder with all the required R packages.
+
+### Make virtual python enviroment
+```bash
+python3 -m venv venv
+source venv/bin/activate
```
Comment (melferink) [inline ExomeDepth/README.md:10]: pip install --upgrade pip
easy_install pip

Comment (rernst) [reply ExomeDepth/README.md:10]: fixed in 6c8998d

File: ExomeDepth/README.md [hunk 2]
```diff
@@ -1,14 +1,32 @@
...
+pip install --upgrade pip
 pip install -r requirements.txt
 ```
 
+### Install R packages in custom library location
+Start a singularity shell with the [rocker tidyverse](https://hub.docker.com/layers/rocker/tidyverse/3.5.1/images/sha256-916d4e919fbac9ee1f06db7622b4731268dff0cb17998b3db6f629a3e58f73a7?context=explore) docker container, set R_LIBS location and open R.
+```bash
+singularity shell -B /path:/path docker://rocker/tidyverse:3.5.1
```
Comment (melferink) [inline ExomeDepth/README.md:18]: path aanduiden als variabele?

File: ExomeDepth/settings.py [hunk 3]
```diff
@@ -18,7 +18,9 @@
 reference_genome = "/hpc/diaggen/data/databases/ref_genomes/Homo_sapiens.GRCh37.GATK.illumina/Homo_sapiens.GRCh37.GATK.illumina.fasta"
 
 #Used R version
-r_version = "R/3.5.1"
+r_library_path = "/data/software/tools/R_libs/3.5.1"
```
Comment (melferink) [inline ExomeDepth/settings.py:21]: Uiteindelijk als default locatie wel hpc path hier neerzetten.
/hpc/diaggen/software/production/R_libs/ExomeDepthv2.0.0/3.5.1_singularity/

**Review by melferink — CHANGES_REQUESTED** [review]
Two minor changes

**Review by melferink — APPROVED** [review]
All changes ok.
Template.vcf is checked manually by diff with original Template.vcf.

File: ExomeDepth/README.md (hunk 2 above)
Comment (rernst) [reply ExomeDepth/README.md]: Done in 9d91b96

File: ExomeDepth/settings.py (hunk 3 above)
Comment (rernst) [reply ExomeDepth/settings.py]: Done in 6c03a2a

**Review by melferink — APPROVED** [review]
All ok. 
Template.vcf checked manually with diff on current production version.

**Pull request merged**